src/components/NoteDetails/NoteDetailsStyles.js

```diff
@@ -17,6 +18,7 @@ export const DefaultTextarea = styled.textarea`
   border-radius: 4px;
   outline: none;
   box-shadow: none;
+  margin-bottom: 20px;
```

Comment:
In the design the distance between the note name and note content is `16px`:

![image](https://user-images.githubusercontent.com/6686679/115890412-94c84a00-a45d-11eb-84a1-9d2c9b9ed49d.png)

Why did you add a `20px` margin ?

Reply:
And in this task you're working on the topbar styles. Why do you even need to change the `textarea` styles?